src/Analyzer.Utilities/AbstractVersionCheckAnalyzer.cs

```diff
@@ -12,7 +12,7 @@ namespace Analyzer.Utilities
     public abstract class AbstractVersionCheckAnalyzer : DiagnosticAnalyzer
     {
         private const string RuleId = "CA9999";
-        private const string AnalyzerPackageVersion = "2.3.*";
+        private const string AnalyzerPackageVersion = "2.5.*";
```

Comment:
@mavasani ? 

I think it will work if you consume 2.6 version or 2.5 version of Roslyn package.